src/olympia/reviewers/views.py

```diff
@@ -882,6 +882,14 @@ def review(request, addon, channel=None):
         amo.LOG.REMOVE_USER_WITH_ROLE.id]
     user_changes_log = ActivityLog.objects.filter(
         action__in=user_changes_actions, addonlog__addon=addon).order_by('id')
+
+    name_translations = (
+        addon.name.__class__.objects.filter(
+            id=addon.name.id, localized_string__isnull=False).exclude(
+            localized_string='')
+        if addon.name else []
+    )
```

Comment:
Works as a solution.  If `name_translations` was preprocessed though, it could be list of tuples of locales+string, e.g. `[(('en-US', 'en-GB'), 'English Translation'),(('fr',), 'Traduction En Français')]` that would result in a shorter list.

Reply:
https://github.com/mozilla/addons-server/pull/13911/commits/a4eeb3cb75eb9e84513028fe0ad832fd5008dd59 does that in template